strawberry/aiohttp/views.py

```diff
@@ -0,0 +1,106 @@
+import json
+from io import BytesIO
+from pathlib import Path
+from typing import Any, Optional, Type
+
+import aiohttp.web
+from strawberry.file_uploads.data import replace_placeholders_with_files
+from strawberry.http import GraphQLHTTPResponse, process_result
+from strawberry.schema import BaseSchema
+from strawberry.types import ExecutionContext, ExecutionResult
+
+
+class GraphQLView(aiohttp.web.View):
+    graphiql = True
+    schema: Optional[BaseSchema] = None
+
+    async def get_root_value(self) -> Any:
+        return None
+
+    async def get_context(self) -> Any:
+        return {"request": self.request}
```

Comment:
Nitpick: Make these return `object` so we don't lose all type-hinting

Reply:
@BryceBeagle is `object` better than `Any`?

Reply:
Yes. `Any` tells the type checker to just _ignore_ anything that makes use of this value. `object` specifies that the type is so generic that it can be basically anything, _but_ it's still a Python object. This makes it so the type checker insists only methods/attributes/etc that are common to _all_ Python objects are used.